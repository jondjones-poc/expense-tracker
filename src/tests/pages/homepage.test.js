import React from 'react';
import { shallow } from 'enzyme';
import Homepage from '../../pages/homepage';

test('Should render homepage correctly', () => {
    const wrapper = shallow(<Homepage />);
    expect(wrapper).toMatchSnapshot();
});
