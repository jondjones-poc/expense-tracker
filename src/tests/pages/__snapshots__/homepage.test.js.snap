// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render homepage correctly 1`] = `
<div>
  <h1>
    Homepage
  </h1>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
